src/Parser/Rogue/Assassination/Modules/Features/Abilities.js

```diff
@@ -0,0 +1,89 @@
+import SPELLS from 'common/SPELLS';
+import CoreAbilities from 'Parser/Core/Modules/Abilities';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    const combatant = this.combatants.selected;
+    return [
+      {
+        spell: SPELLS.VANISH,
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 120,
+      },
+      {
+        spell: SPELLS.VENDETTA, // TODO: Reduced by Convergence of Fates
+        category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+        cooldown: 180,
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 1.0,
+          extraSuggestion: 'In most cases should be used on cooldown',
```

Comment:
This is repeating what the suggestion already says.  You should be specific about cases where you would hold it.  The goal with the extra suggestion to arm the player with more information they might not already know.